content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -176,9 +179,16 @@ network of the consumers.
 ## Control plane models
 
 An Istio mesh uses the {{< gloss >}}control plane{{< /gloss >}} to configure all
-communication between workload instances within the mesh. You can replicate the
-control plane, and workload instances connect to any control plane instance to
-get their configuration.
+communication between workload instances within the mesh. At one extreme, you can choose
+to deploy one control plane per cluster in the mesh, while at the other extreme,
+you can host one single shared control plane for all clusters in the mesh.
```

Comment:
nit: suggest rephrasing to `you can host one control plane that is shared by all clusters in the mesh`

Reply:
Should we add external control plane here as well?

Reply:
Yeah I think that makes sense. We can probably just add that in a follow-on PR.